### PR TITLE
Item Position Rando Fixes

### LINKED ISSUE #7004: Item Position Shuffle bugs
- Artifact Temple's event to trigger the statues is no longer separated from the item. This means you have to pick up the item to trigger the statues again.

- Artifact Temple used to also have a far more condensed box for the item to spawn in as well due to the shear scale of the box (FSC was left unchanged iirc but the spawning radius of the item in AT was changed similar to how Zoid was.. I don't remember if Sunchamber's items were adjusted too or not to follow the changes)

- Item scans are no longer shown through walls. Before, there was a patch added that showed enlarged scans for the items to better identify where they were in the room, even if they weren't scannable 99% of the time. This made it easier to locate the item and identify what path you had to take to get it.

- Zoid's item on defeat is no longer forced to spawned in the room where Zoid is fought i.e the small circular arena. This used to be the case to make it far less likely to soft lock in zoid given the box of the room.

### PR DESCRIPTION
**Pull request by toasterparty**
It's a largely unused chaos option but it's fun so I fix

Fixes #7004

(no changelog because experimental option)

### CODE REVIEW

**Comment by Miepee** [comment]
> (no changelog because experimental option)

The option is in stable tho, so it affects the next release and it affects gameplay. Looking at our contributions.md, I cant see any mention where changelog for experimental options should be skipped, only experimental games as they dont make it into a stable release.
Would be neat to collectively decide on how to handle that, cause i personally lean more towards also specifying a changelog for these.


Also your tests are failing.

File: CHANGELOG.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed: Black bars sometimes disappearing prematurely in the Chozo Ice Temple cutscene
 - Fixed: Missing screen fade-in from Arrival cutscenes in Transport to Chozo Ruins East and Transport to Tallon Overworld South when skipped
 - Fixed: Ghost music in Furnace not playing when coming from Energy Core
+- Fixed: [Experimental] Item Position Rando fixes in Artifact Temple and Burn Dome
```
Comment (Miepee) [inline CHANGELOG.md:94]: This is failing to mention the jumbo scan through walls thing.

Comment (toasterparty) [reply CHANGELOG.md:94]: eh

Comment (Miepee) [reply CHANGELOG.md:94]: i mean, it is part of the mentioned issue and also getting fixed.

**Review by Miepee — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -91,6 +91,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed: Black bars sometimes disappearing prematurely in the Chozo Ice Temple cutscene
 - Fixed: Missing screen fade-in from Arrival cutscenes in Transport to Chozo Ruins East and Transport to Tallon Overworld South when skipped
 - Fixed: Ghost music in Furnace not playing when coming from Energy Core
+- Fixed: [Experimental] Item Position Rando fixes: Scan visibility through walls and Artifact Temple/Burn Dome positions
```
Comment (Miepee) [inline CHANGELOG.md:94]: changelog entry needs to be in new version now.

**Review by Miepee — CHANGES_REQUESTED** [review]

**Review by Miepee — APPROVED** [review]

**Pull request merged**